test/runnable/test28.d

```diff
@@ -1247,6 +1247,49 @@ void test65()
 }
 
 
+/*******************************************/
+// https://issues.dlang.org/show_bug.cgi?id=18576
+
+void delegate() callback;
+
+struct S66 {
+    int x;
+    @disable this(this);
+
+    this(int x) {
+        this.x = x;
+        callback = &inc;
+    }
+    void inc() {
+        x++;
+    }
+}
+
+auto f66()
+{
+    return g66();   // RVO should be done
+}
+
+auto g66()
+{
+    return h66();   // RVO should be done
+}
+
+auto h66()
+{
+    return S66(100);
+}
+
+void test18576()
+{
+    auto s = f66();
+    printf("%p vs %p\n", &s, callback.ptr);
+    callback();
+    printf("s.x = %d\n", s.x);
```

Comment:
`printf` leftovers in tests.

Reply:
They're not a problem. Lots of tests in the test suite are chatty.